labrad/units.py

```diff
@@ -679,7 +679,7 @@ def base_unit(self):
         for unit, power in (zip(_base_names, self.powers) +
                             self.lex_names.items()):
             if power != 1 and power != -1:
-                unit += '**' + str(abs(power))
```

Comment:
`**` should work in python code, since that's python's exponentiation operator; but it's not correct in the string representation of units.